colossalai/utils/device.py

```diff
@@ -44,6 +44,18 @@ def get_current_device() -> torch.device:
         return torch.device("cpu")
 
 
+def get_current_device_id() -> int:
```

Comment:
It seems this is similar to `current_device()` in this file

Reply:
Yes, it returens "int" and current_device returns "device". Device index is required sometimes.

Reply:
`current_device()` returns int, and `get_current_device()` returns device

Reply:
ok, i have removed it